packages/web-components/src/components/card/__stories__/card.stories.ts

```diff
@@ -101,9 +101,9 @@ export default {
   title: 'Components/Card',
   decorators: [
     story => html`
-      <div class="dds-ce-demo-devenv--simple-grid dds-ce-demo-devenv--simple-grid--card">
-        <div class="bx--row dds-ce-demo-devenv--grid-row">
-          <div class="bx--col-sm-4 bx--col-md-3 bx--col-lg-12 bx--col-xlg-8 bx--no-gutter">
+      <div class="bx--grid dds-ce-demo-devenv--grid--stretch">
+        <div class="bx--row">
+          <div class="bx--col-sm-4 bx--col-md-3 bx--col-lg-6 bx--col-xlg-4 bx--no-gutter">
```

Comment:
Actually the codebase is transitioning away from `bx--grid` class as part of the effort described in #4174 epic. That said, would you try updating the CSS rulesets for `.dde-ce-demo-devenv--simple-grid--card`? You can find them at: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.15.0-rc.2/packages/web-components/.storybook/container.scss#L77-L101

Similar thing applies to card link story. Thanks!